packages/react-native-worklets/Common/cpp/worklets/Tools/AsyncQueue.cpp

```diff
@@ -49,4 +49,10 @@ void AsyncQueue::push(std::function<void()> &&job) {
   state_->cv.notify_one();
 }
 
+void AsyncQueue::empty() {
```

Comment:
I would expect `AsyncQueue::empty` method to check whether the data structure is empty rather than making it empty. Can we rename it to `terminate` or `shutdown` instead?